File: Java/programmers/etc/level1/K번째수.java
package CodeTest.Java.programmers.etc.level1;

import java.util.*;

public class K번째수 {
    public int[] solution(int[] array, int[][] commands) {
        int n = commands.length;
        int[] answer = new int[n];
        
        for (int i=0; i<n; i++) {
            int[] tmp = Arrays.copyOfRange(array, commands[i][0]-1, commands[i][1]);
            Arrays.sort(tmp);
            answer[i] = tmp[commands[i][2]-1];
        }

        return answer;
    }
}

File: Python/programmers/etc/level1/K번째수.py
def solution(array, commands):
    answer = []
    for a, b, c in commands:
        answer.append(sorted(array[a-1:b])[c-1])
    return answer


array = [1, 5, 2, 6, 3, 7, 4]
commands = [[2, 5, 3], [4, 4, 1], [1, 7, 3]]
print(solution(array, commands)) # [5, 6, 3]
